cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -396,7 +397,14 @@ private Future<Void> reconcile(Reconciliation reconciliation, String host,
                                    CruiseControlApi apiClient, KafkaRebalance kafkaRebalance,
                                    KafkaRebalanceState currentState, KafkaRebalanceAnnotation rebalanceAnnotation) {
 
-        LOGGER.infoCr(reconciliation, "Rebalance action from state [{}]", currentState);
+
+        if (kafkaRebalance != null && kafkaRebalance.getStatus() != null
+                && Objects.equals(rebalanceStateConditionType(kafkaRebalance.getStatus()), "Ready")
+                && rawRebalanceAnnotation(kafkaRebalance) == null) {
+            LOGGER.infoCr(reconciliation, "Rebalancing is completed. You can use the  `refresh` annotation to ask for a new rebalance request", currentState);
```

Comment:
The double-space is a typo ... do you need the current status? There does not seem to be any placeholder for it in the log message.

```suggestion
            LOGGER.infoCr(reconciliation, "Rebalancing is completed. You can use the `refresh` annotation to ask for a new rebalance request");
```